docs/UserGuide.md

```diff
@@ -514,24 +552,26 @@ Therefore, to change the status of leave, please use the <a href= "#approve-leav
 | `Date should be valid and in a format of `yyyy-MM-dd`` | The start date and end date inputs are not in the correct format | Ensure that the inputs of dates are in the format of `yyyy-MM-dd`, for example, `2023-01-01`. |
 | `Leave descriptions should only contain alphanumeric characters, spaces, dashes, commas, apostrophes and full stops.` | Description input contains illegal characters | Remove the illegal characters from the input.
 
-### Find a leave record
+<div style="page-break-after: always;"></div>
+
+### Find a leave application
 
-HRMate offers different commands for finding leave records. [`find-leave-range`](#find-leave-records-by-time-period-find-leave-range) and [`find-leave-status`](#find-leave-records-by-leave-status-find-leave-status) would find leave records by time period and leave status respectively while [`find-leave`](#find-leave-records-belonging-to-an-employee-find-leave) and [`find-all-leave`](#view-all-leaves-find-all-leave) would find leave records by employee and view all leave records respectively.
+HRMate offers different commands for finding leave applications. [`find-leave-range`](#find-leave-records-by-time-period-find-leave-range) and [`find-leave-status`](#find-leave-records-by-leave-status-find-leave-status) would find leave applications by time period and leave status respectively while [`find-leave`](#find-leave-records-belonging-to-an-employee-find-leave) and [`find-all-leave`](#view-all-leaves-find-all-leave) would find leave applications by employee and view all leave applications respectively.
```

Comment:
can check if these links work, because they still say leave records while we already renamed them to leave applications